File: Jour12/jour12.c
#include<stdio.h>

int main()
{
	int n,i;
	printf(" =================== Ce programme affiche la table de multiplication d'un nombre de 1 a 10 ===================\n");
	do
	{
		printf("Donner un nombre entier : ");
		scanf("%d",&n);
		if(n != (int)n)
			printf("Erreur :::: ressayer.\n");
	}while(n != (int)n);
	printf("TABLE DE MULTIPLICATION PAR : %d\n",n);
	for(i = 1; i <= 10; i++)
		printf("%d * %d = %d\n",n,i,n*i);
	return 0;
}

File: Jour12/jour12.py
#coding:utf-8
print(" =================== Ce programme affiche la table de multiplication d'un nombre de 1 a 10 ===================")
while 1:
	n = input("Donner un nomnbre entier : ")
	try:
		n = int(n)
	except:
		print("Saisie invalide :::: ressayer")
		continue
	else:
		break
print(f"TABLE DE MULTIPLICATION PAR : {n}")
for i in range(10):
	print(f"{n} * {i+1} = {n*(i+1)}")
